docs/javascript/browser-apis-and-jquery.md

```diff
@@ -446,3 +446,308 @@ function increaseCatLikes() {
+        waving_hand
+        </span>
+    <form>
+        <input type="text" placeholder="Enter Wizard Name" id="wizardName" autocomplete="off">
+        <input id="sub" type="submit" value="Submit">
+    </form>
+
+
+    <script>
+
+    function submitName(event) {
+        event.preventDefault();
+        var wizardName = $("#wizardName").val();
+        $("#heading").text(`Welcome ${wizardName}!`);
+    }
+    // add the wizardName to our heading element
+    $("form").submit(submitName);
+
+
+
+    </script>
+
+</div>
+
+</html>
+```
+
+The `change()` method will execute a function when the input value has changed.
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+<title>Events!</title>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+</head>
+
+<body>
+
+<div>
+    <button id="name">Click</button>
+    <h1 id="heading">Welcome!</h1>
+    <span class="material-icons-outlined" id="wave">
+        waving_hand
+        </span>
+    <form>
+        <input type="text" placeholder="Enter Wizard Name" id="wizardName" autocomplete="off">
+        <input id="sub" type="submit" value="Submit">
+    </form>
+
+
+    <script>
+
+   function vanishInput() {
+        $("form").fadeOut();
+    }
+    // vanish form once the input value has changed
+    $("form").change(vanishInput);
+
+
+    </script>
+
+</div>
+
+</html>
+
+```
+
+The `mouseover()` method allows us to execute a function when the mouse is over the element.
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+<title>Events!</title>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+</head>
+
+<body>
+
+<div>
+    <button id="name">Click</button>
+    <h1 id="heading">Welcome!</h1>
+    <span class="material-icons-outlined" id="wave">
+        waving_hand
+        </span>
+    <form>
+        <input type="text" placeholder="Enter Wizard Name" id="wizardName" autocomplete="off">
+        <input id="sub" type="submit" value="Submit">
+    </form>
+
+
+    <script>
+
+    function logoChange() {
+        $("#wave").animate({ fontSize: "100px",}, "2s");
+    }
+    //change logo size once mouse is over the element
+    $("#wave").mouseover(logoChange);
+
+
+    </script>
+
+</div>
+
+</html>
+```
+
+The `mousemove()` method allows us to execute a function when the mouse moves on the given element.
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+<title>Events!</title>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+</head>
+
+<body>
+
+<div>
+    <button id="name">Click</button>
+    <h1 id="heading">Welcome!</h1>
+    <span class="material-icons-outlined" id="wave">
+        waving_hand
+        </span>
+    <form>
+        <input type="text" placeholder="Enter Wizard Name" id="wizardName" autocomplete="off">
+        <input id="sub" type="submit" value="Submit">
+    </form>
+
+
+    <script>
+
+    function logoChangeTwo() {
+        $("#wave").css({color: "blue"});
```

Comment:
I'm going to try to enforce this as I build new courses, but let's go ahead and start doing it in the docs. No more POJOs as args to functions. We'll do this from now on:

```javascript
function logoChangeTwo() {
    
    var changeLogoCSS = {
        color: "blue",
    };

    $("#wave").css(changeLogoCSS);
}
```